src/Sarif.Multitool/MergeCommand.cs

```diff
@@ -59,9 +58,13 @@ private static IEnumerable<Run> GetAllRuns(IEnumerable<string> sarifFiles)
 		    }
 	    }
 
-	    private static IEnumerable<string> GetSarifFiles(IEnumerable<string> files)
+	    private static IEnumerable<string> GetSarifFiles(MergeOptions mergeOptions)
 	    {
-		    foreach (var path in files)
+		    var searchOption = mergeOptions.Recursive 
+				? SearchOption.AllDirectories 
+				: SearchOption.TopDirectoryOnly;
+
+		    foreach (var path in mergeOptions.Files)
```

Comment:
>var [](start = 15, length = 3)

same note, use 'string' not var here

Reply:
Done.

---
In reply to: [141649387](https://github.com/Microsoft/sarif-sdk/pull/627#discussion_r141649387) [](ancestors = 141649387)